Explorer/Assets/DCL/PerformanceAndDiagnostics/Profiling/ECS/DebugViewProfilingSystem.cs

```diff
@@ -84,10 +105,24 @@ protected override void Update(float t)
                     UpdateFrameStatisticsView(profiler);
                 }
 
+                if (frameTimingsEnabled && bottleneckDetector.TryCapture())
```

Comment:
According to the docs, [FrameTimingManager](https://docs.unity3d.com/ScriptReference/FrameTimingManager.html) is not always available. Maybe we can add an extra check for this to avoid possible confussions?
Just a call to [IsFeatureEnabled](https://docs.unity3d.com/ScriptReference/FrameTimingManager.IsFeatureEnabled.html)